scanner/scanner.go

```diff
@@ -377,14 +379,28 @@ func (s *Scanner) Scan(foundCert func(*ct.LogEntry),
 	fetches := make(chan fetchRange, 1000)
 	jobs := make(chan matcherJob, 100000)
 	go func() {
+		slidingWindow := make([]int64, 15)
+		i, previousCount := 0, int64(0)
 		for range ticker.C {
 			certsProcessed := atomic.LoadInt64(&s.certsProcessed)
-			throughput := float64(certsProcessed) / time.Since(startTime).Seconds()
+			slidingWindow[i%15], previousCount = certsProcessed-previousCount, certsProcessed
```

Comment:
Disagree, I'd rather have these 10, reasonably readable, not reused lines in here rather than hidden in a type that the reader would have to jump to anyway to understand the semantics of.

(But of course, your repo, so feel free to override.)